src/site/content/en/blog/intrinsic-layout-overview/index.md

```diff
@@ -0,0 +1,151 @@
+{% endAside %}
+
+A way I like to think of it is self-reflective. You—yes you the reader—if you
+stand up with no shoes on, what's your height? What's your width? **Those
+numbers make up your intrinsic size!** Extend your arms to the ceiling—that's
+your `max-content`. Crouch down—that's your `min-content`. Those sizes are
+relative to you, and they're quite personal.
+
+Now think about a friend or family member, and think about their natural
+`height`, `width`, `max-content` and `min-content`. It's probably different from
+yours.
+
+**Intrinsic design means treating your content just as you would a fellow human
+that's dynamic, different, and unique.** It means creating layouts that are
+respectful of your content's needs so as not to squish something beyond its
+`min-content` or extend it beyond its `max-content`.
+
+It also means accomodating those needs if they change. To do this we need to
+leverage every part of the CSS toolbox and choose the right tool for the job. As
+Jen explains:
+
+<blockquote class="w-blockquote">
+  <p class="w-blockquote__text">
+    It includes CSS Grid, but it’s not just about CSS Grid. It’s also about using
+    Flexbox, and kind of rediscovering what Flexbox is actually intended to be for.
+    Plus, it’s about using some floats sometimes, using things like CSS shapes or
+    object-fit, using a flow content, using multi-column. Some of these things are
+    old, and they’ve been around for a long time, but it’s about thinking about the
+    whole system of layout, and how all these pieces fit together in a brand new
+    way.
+  </p>
+  <cite class="w-blockquote__cite">
+    <a href="http://www.zeldman.com/2018/05/02/transcript-intrinsic-web-design-with-jen-simmons-the-big-web-show/">Intrinsic web design with Jen Simmons</a>
+  </cite>
+</blockquote>
+
+## Why "intrinsic"? 🤔
+This may look familiar, it's the result of an _extrinsic_ approach to layout:
+
+<p class="codepen" data-height="401" data-theme-id="dark" data-default-tab="result" data-user="brundolf" data-slug-hash="gRaREv" style="height: 401px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="CSS is Awesome">
+  <span>See the Pen <a href="https://codepen.io/brundolf/pen/gRaREv/">
+  CSS is Awesome</a> by Brandon (<a href="https://codepen.io/brundolf">@brundolf</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
+The article [CSS is Awesome](https://css-tricks.com/css-is-awesome/) recaps the
+frustration of this classic CSS joke really well:
+- The content doesn't shrink to fit the container
+- The container doesn't expand to fit the content
+- The container doesn't handle overflow gracefully
+
+In my experience, initially laying out a site extrinsicly is the easiest. Later,
+it becomes tedious to maintain or even becomes a problem for other folks on the
```

Comment:
Suggestion to change 'a problem for other folks on the team that aren't engineers' to 'a problem for team members who try and work on the site later on'.  Just a little wary of the 'not engineer part'.